src/sentry/api/serializers/models/organization_member/expand/roles.py

```diff
@@ -15,11 +14,17 @@
 
 
 def _is_retired_role_hidden(role: OrganizationRole, member: OrganizationMember) -> bool:
+    """
+    During EA, we will show the role but make it greyed out and prevent the user
```

Comment:
Does this helper determine whether it's hidden or greyed out?

Reply:
Yea it doesn't pass the role to the frontend, so it doesn't even appear at all.